CHANGELOG.md

```diff
@@ -3,11 +3,12 @@
 ## Upcoming
 
 - `apollo`
+  - Improve performance of CLI when running projects by delaying or not calling validation [#1559](https://github.com/apollographql/apollo-tooling/pull/1559)
```

Comment:
This change note sounds kind of scary. Can we add the word "unnecessarily" at the end to make it obvious that we're omitting validation when when didn't need it?

Reply:
Good thought @lilyball! Updated